### PR TITLE
DOC: Miscellaneous tiny changes

### PR DESCRIPTION
**Pull request by j-t-1**

### CODE REVIEW

File: docs/user/cropping-and-transforming.md [hunk 1]
```diff
@@ -25,8 +25,6 @@ page3.mediabox.upper_right = (
 writer.add_page(page3)
 
 # Add some JavaScript to launch the print window on opening this PDF.
-# The password dialog may prevent the print dialog from being shown,
```
Comment (stefan6419846) [inline docs/user/cropping-and-transforming.md:28]: Why can we remove this?

File: docs/conf.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 # -- Project information -----------------------------------------------------
 
 project = py_pkg.__name__
-copyright = "2006 - 2023, Mathieu Fenniak and pypdf contributors"
+copyright = "2006 - 2024, Mathieu Fenniak and pypdf contributors"
```
Comment (stefan6419846) [inline docs/conf.py:28]: I recommend generating this on-the-fly instead.

Comment (j-t-1) [reply docs/conf.py:28]: Good idea did not know this was possible.

File: docs/user/cropping-and-transforming.md (hunk 1 above)
Comment (j-t-1) [reply docs/user/cropping-and-transforming.md:28]: This JavaScript distracts from the example, and I wanted to minimize it. Could be removed but is the only `add_js` example in the documentation.

Comment (stefan6419846) [reply docs/user/cropping-and-transforming.md:28]: Please either keep this here or add a dedicated example at an appropriate location instead.

Comment (j-t-1) [reply docs/user/cropping-and-transforming.md]: Reverted.

**Review by stefan6419846 — APPROVED** [review]

**Pull request merged**